src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -286,7 +286,7 @@ private static boolean isValid(VendorList vendorList) {
      * Handles errors occurred while HTTP request or response processing.
      */
     private static Future<Void> failResponse(Throwable exception, int version) {
-        logger.warn("Error fetching vendor list via HTTP for version {0}", exception, version);
+        logger.warn("Error fetching vendor list via HTTP for version {0}", version);
```

Comment:
i'd like to see more details when error occurred.
Also, pls look at PR https://github.com/rubicon-project/prebid-server-java/pull/574 which adds INFO error level logging.

Reply:
Ok